tests/cbmc/proofs/s2n_set_free/s2n_set_free_harness.c

```diff
@@ -21,10 +21,24 @@ void s2n_set_free_harness()
 {
     /* Non-deterministic inputs. */
     struct s2n_set *set = cbmc_allocate_s2n_set();
-    __CPROVER_assume(s2n_result_is_ok(s2n_set_validate(set)));
 
+    /* Assumptions. */
     nondet_s2n_mem_init();
+    __CPROVER_assume(s2n_result_is_ok(s2n_set_validate(set)));
 
     /* Operation under verification. */
-    s2n_set_free(set);
+    s2n_result result = s2n_set_free(set);
+    if (s2n_result_is_error(result)) {
+        assert(s2n_errno != S2N_ERR_FREE_STATIC_BLOB);
+    }
+
+    /* Cleanup after expected error cases, for memory leak check. */
+    if (s2n_result_is_error(result)) {
```

Comment:
You can merge these conditions together.

Reply:
Merged as suggested. Thanks!